stats/view/view.go

```diff
@@ -28,112 +27,109 @@ import (
 	"go.opencensus.io/tag"
 )
 
-// View allows users to filter and aggregate the recorded events.
-// Each view has to be registered to enable data retrieval. Use New to
-// initiate new views. Unregister views once you don't want to collect any more
-// events.
+// View allows users to aggregate the recorded stats.Measurements.
+// Views need to be passed to the Subscribe function to be before data will be
+// collected and sent to Exporters.
 type View struct {
-	name        string // name of View. Must be unique.
-	description string
+	Name        string // Name of View. Must be unique.
+	Description string // Description is a human-readable description for this view.
 
-	// tagKeys to perform the aggregation on.
-	tagKeys []tag.Key
+	// GroupByTags are the tag keys describing the grouping of this view.
+	// A single Row will be produced for each combination of associated tag values.
+	GroupByTags []tag.Key
 
-	// Examples of measures are cpu:tickCount, diskio:time...
-	m stats.Measure
+	// MeasureName is the name of the stats.Measure to aggregate in this view.
+	MeasureName string
```

Comment:
Let's still use Measure, I don't see the utility of the user invoking m.Name() and then when registering, we also look for the Measure by name. In fact exposing this field as a name `MeasureName` means that some users will accidentally not create measures assuming we'll automagically do it for them, at least I'd fall for this.

Reply:
Good point. I was wanting to move this further in the direction of allowing views to be defined as configuration rather than code but that is really a separate feature so I'll revert to Measure.

Reply:
Ditto.